src/index.html

```diff
@@ -8,6 +8,76 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+
```

Comment:
You don't need spaces here, please check 1 point in the checklist again :)

Reply:
oh no I understand now, yes I missed an important thing, sorry